### PR TITLE
Implement useSSL connection config option

### PR DESCRIPTION
**Pull request by rsolano**
Implements the `useSSL` configuration option available via the [Java ArangoDB driver](https://www.arangodb.com/docs/stable/drivers/java-reference-setup.html).

### CODE REVIEW

**Review by BigMatt — APPROVED** [review]

**Comment by Azolo** [comment]
I think you should fork the repo personally.

File: pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
   <groupId>io.github.jaredpetersen</groupId>
   <artifactId>kafka-connect-arangodb</artifactId>
-  <version>1.0.6</version>
+  <version>1.0.7</version>
```
Comment (Azolo) [inline pom.xml:8]: I wouldn't bump this up without a sem-var build or pre-release modifier. Since it's not an official release.

**Review by Azolo — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (Azolo) [reply pom.xml:8]: https://semver.org/#backusnaur-form-grammar-for-valid-semver-versions

Comment (rsolano) [reply pom.xml:8]: I did this for the official release, changeset was merged a while ago into the original project.

**Pull request merged**